### PR TITLE
Update travis settings

### PR DESCRIPTION
**Pull request by matthiasbeyer**
Travis changes

Ref #14

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,23 @@
...
-    - stable
+  - 1.17.0
+  - 1.18.0
+  - stable
+  - beta
+  - nightly
+
+matrix:
+  allow_failures:
+    - rust: nightly
```
Comment (untitaker) [inline .travis.yml]: Why this?

Comment (matthiasbeyer) [reply .travis.yml]: Well it is the default I always use... I try not specifically to support nightly rust but it is nice to know when something is about to fail in the upcoming beta...

We could either remove nightly from the build jobs, allow failures there or build with nightly in mind, what would you suggest?

Comment (untitaker) [reply .travis.yml]: I probably won't look at the details of a build that already has a green indicator on GitHub, so I don't think I would notice if anything happens with this build.

Comment (matthiasbeyer) [reply .travis.yml]: Okay, so I will remove nightly from the chain.

I will also update the rust versions we build with, I guess...

**Comment by matthiasbeyer** [comment]
If you're fine with the patchset, I will rebase-squash the changes :+1:

**Comment by untitaker** [comment]
LGTM, thanks! I can squash from the web UI.

**Pull request merged**